free-1.md

```diff
@@ -56,13 +64,16 @@ spent directly engaging in the **language** aspects of this project (e.g.,
 making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
+I think it would be a little more system aspects because I would have to figure out what words mean what and in what contexts while the english is pretty standardized already.
 
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
+I think this is a medium sized project. It would be pretty ambitious and one thing I would be worried about would be integration with a computers calendar, but I think it is doable.
 
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
+I think this is a good idea because it sounds really cool, fun and interesting but it might be a bad idea because, as ive been doing this, Ive realized that its only kind of a DSL. The language here is english and the service is mostly parsing?
```

Comment:
I agree that this is a cool idea and that, as stated, it might be more of a Natural-Language Processing project. There is at least one "DSL" for calendars: [ICalendar](https://en.wikipedia.org/wiki/ICalendar), which formats calendar events. The project could translate natural language into ICalendar; however that would be more about natural language processing than about language design.